ui/pages/Login/reducers.js

```diff
@@ -19,6 +20,42 @@ export const login = (values) => {
   }
 }
 
+export const googleLogin = () => {
+  return new HttpRequestHelper('/api/login_google',
+    (responseJson) => {
+      // Redirect to google auth website
+      const width = 600
```

Comment:
these should be top-level constants in all caps

Reply:
Got it. But I may not change it for now. If I separate front end React UI from the authentication flow, this page will become a Django View page.